client/components/ProgressModules.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ProgressModule from './ProgressModule'
+
+export default function ProgressModules (props) {
+  const { modules, assessments, studentAssessments } = props
+  const progressModules = modules.map(module => {
+    return <ProgressModule key={module.id} progress={calculateProgress(module.id, assessments, studentAssessments)}/>
```

Comment:
Could we break this up into multiple lines and not one long line?
```
return <ProgressModule
key={}
progress= {}
```

Reply:
Absolutely - working on this now.